src/porepy/numerics/fv/mpsa.py

```diff
@@ -506,7 +506,7 @@ def assemble_matrix_rhs(
         stress = matrix_dictionary["stress"]
         bound_stress = matrix_dictionary["bound_stress"]
 
-        div = pp.fvutils.vector_divergence(sd)
+        div = sd.divergence(dim=2)
```

Comment:
I also stumbled upon this. We need to define "our" convention, what `dim` means. The dimension of the linear algebra object (1-dim is vector, 2-dim is matrix, 3-dim is 3-tensor etc). Then dim=2 simply means vector divergence, and dim=3 actually never is used in PP contexts. See also my comment directly on the documentation of the definition of the divergence operator.

Reply:
Updated to sd.dim

Note that this is a method that only survives because it is used in the tutorial on stress dicsretizations (from what I can tell). Unfortunately, 1 >> 0.

@jwboth answer in two (1.5 parts): 
1. We are trying to introduce the notion that dim is used for genal dimensions, while nd specifically represents the ambient spatial dimension. Roll-out is gradual, to say the least. 
2. The dimension, in this context, always refers to the number of quantities per face/cell/etc. Considering the generality of the concept of dimensions, I don't think we can set a fixed rule for what we mean by it.